### ISSUE TITLE
Perfectionist sorting rules have incorrect kebab-case rules

### ISSUE THREAD

**Issue by rwarren** [EvgenyOrekhov/eslint-config-hardcore#950]
## Problem description

Current perfectionist sorting rules are here (and below):
[sort-array-includes](https://github.com/EvgenyOrekhov/eslint-config-hardcore/blob/06b0e1d40fe5fa13cea7416ecebaf683c52a83e7/base.json#L847)

They currently use kebab-case for `ignore-case`, like this:
```javascript
    "perfectionist/sort-array-includes": [
      "error",
      { "type": "natural", "ignore-case": true }
    ],
```

This results in errors like this with ESLint:
```
[Error - 6:07:49 PM] An unexpected error occurred:
[Error - 6:07:49 PM] Error: Key "rules": Key "perfectionist/sort-enums":
	Value {"type":"natural","ignore-case":true} should NOT have additional properties.
		Unexpected property "ignore-case". Expected properties: "fallbackSort", "specialCharacters", "ignoreCase", "alphabet", "locales", "order", "type", "forceNumericSort", "customGroups", "sortByValue", "partitionByComment", "partitionByNewLine", "newlinesBetween", "groups".
```

The problem is that the options are camelCase, not kebab-case. Note that it should be `ignoreCase` and not `ignore-case`.

Perfectionist must have switched?! Unclear why this is wrong and couldn't immediately see a switchover in the Perfectionist git blame.

## Workaround

To work around this you can just override the rules with proper casing.

**Comment by EvgenyOrekhov** [comment]
You must have a later version of Perfectionist for some reason. In any case, this won't be a problem in the next release.

**Comment by EvgenyOrekhov** [comment]
Fixed in https://github.com/EvgenyOrekhov/eslint-config-hardcore/commit/2902c031020c9151ec826577c841ff563fb2750e
Released in [v49.0.0](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v49.0.0)
